drivers/docker/handle.go

```diff
@@ -216,11 +217,15 @@ func (h *taskHandle) shutdownLogger() {
 		return
 	}
 
-	if err := h.dlogger.Stop(); err != nil {
-		h.logger.Error("failed to stop docker logger process during StopTask",
-			"error", err, "logger_pid", h.dloggerPluginClient.ReattachConfig().Pid)
-	}
-	h.dloggerPluginClient.Kill()
+	go func() {
+		h.logger.Info("sending stop signal to logger", "job_id", h.task.JobID, "container_id", h.containerID)
```

Comment:
Does this need to be info level?

```suggestion
		h.logger.Debug("sending stop signal to logger", "job_id", h.task.JobID, "container_id", h.containerID)
```